src/datasources/balances-api/valk-balances-api.service.ts

```diff
@@ -93,6 +95,14 @@ export class ValkBalancesApi implements IBalancesApi {
     await this.cacheService.deleteByKey(key);
   }
 
+  async getCollectibles(): Promise<Page<Collectible>> {
+    throw new Error('Method not implemented.');
+  }
+
+  async clearCollectibles(): Promise<void> {
+    throw new Error('Method not implemented.');
+  }
```

Comment:
Do you think we should already remove Valk?

Reply:
Unfortunately, given the state of the code when this PR was done when adding these methods to the common interface, we need to "implement" them here. But this file is being removed in this follow-up PR: https://github.com/safe-global/safe-client-gateway/pull/1149/files#diff-5ed3bd882f584f2b5e7ce858bdaa913df6e736ce24e12d45608850ce22c8340c